index.markdown

```diff
@@ -682,7 +683,8 @@ have any concerns, you should seek a professional.
 Some people have had luck with this
 [free government program][IRS-FreeTaxReturn].
 
-CECA has [some information][CECA-USIncomeTax] on taxes.
+CECA has [some information][CECA-USIncomeTax] on taxes. Approximately one year after completion of a J-1 Visa on the Cultural Vistas program offered by the CECA, students are offered a free code from [Glacier Tax Prep](https://www.glaciertax.com/) to do their taxes with.
```

Comment:
Is it one year after or just before tax season?

I ask because if someone is on a Fall internship will they get the code in time for April?